### PR TITLE
Enable special color when force-push is needed

### PR DESCRIPTION
**Pull request by destogl**
What do you think?

It looks like this:

slika

### CODE REVIEW

**Comment by gwalck** [comment]
green (or grey, not sure) text on orange is not so readable. Complementary color to orange is blue. maybe would be more visible ?

**Comment by destogl** [comment]
OK, this is not properly visible in any combination.

Let's go with brown/orange color like this:

slika

File: scripts/configuration/terminal_coloring.bash [hunk 1]
```diff
@@ -75,14 +75,15 @@ function set_git_color {
...
+    elif [[ "$STATUS" == *'Your branch is ahead'* ]]; then
         # yellow if need to push
         color=${TERMINAL_COLOR_YELLOW}
-      else
-        # else green
-        color=${TERMINAL_COLOR_GREEN}
-      fi
+    elif [[ "$STATUS" == *' have diverged,'* ]]; then
+      # brown if need to force push
+      color=${TERMINAL_COLOR_BLACK}${TERMINAL_BG_COLOR_YELLOW}
```
Comment (destogl) [inline scripts/configuration/terminal_coloring.bash:83]:
```suggestion
      color=${TERMINAL_COLOR_BROWN}
```

**Comment by gwalck** [comment]
looks better indeed.

**Pull request merged**